examples/with-indent/pages/_app.js

```diff
@@ -0,0 +1,27 @@
+import App from 'next/app'
+import Router from 'next/router'
+import audit from '@indent/audit'
+
+audit.init({
+  // Find your DSN on Indent Docs https://indent.com/docs/quickstart
+  dsn: '{{INDENT_INPUT_DSN}}',
```

Comment:
If this variable is required for the example to work properly, add a `now.json` that requires it as a secret, so when trying to deploy using the deployment button in the example, ZEIT Now will ask for the secret before making the deployment.